annalieseschropp/checkers
# 9.1_RecordsBackend

What was Accomplished

- The Leaderboard scene of the project was created in the LeaderBoard scene.
- The Leaderboard scene includes:
     - A table to display the leaderboard containing all player names and the amount of games they've won and lost.
- Created the backend for the record keeper using a static class to it may be accessed from anywhere in the application.
- Unit Testing, passes all 7 tests
    -  Testing all possible paths for the record keeper to happen with.

Testing

    File > Build Settings > Add Open Scenes > Build
    You also need the Test Runner window open to see the available tests
    Window > General > Test Runner
    You can dock this where you would like based on your layout preferences
    Click Run All in the Test Runner window or select a specific test then click Run Selected
    To manually run the scene, press the play button or go to File > Build and Run

To Dos:

- Connect the scene to the main menu
- Add ability for records to be added from name entry menu.
- Update records based on game winning and losing of players.
